examples/mobilenet/mobilenet_qnn_ep_template.json

```diff
@@ -6,13 +6,24 @@
         }
     },
     "systems": {
+        "local_system": {
+            "type": "LocalSystem",
+            "config": {
+                "accelerators": [
+                    {
+                        "execution_providers": [
+                            "QNNExecutionProvider"
```

Comment:
it seems the qnn ep is installed in both local and python system. what would happens if we change skip_supported_eps_check=False or remove this flag?

Reply:
I suppose the flag is only used if the target ep is not installed with the related ORT?